apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.ValueConversion.cs

```diff
@@ -393,8 +401,8 @@ private object ConvertToClrTypeImpl(Value wireValue, System.Type targetClrType)
                     case Value.KindOneofCase.StructValue:
                         foreach (var structField in StructMembers)
                         {
-                            dictionary[structField.Key] = structField.Value.ConvertToClrType(
-                                    wireValue.StructValue.Fields[structField.Key], itemType);
+                            var fieldValue = wireValue.StructValue.Fields[structField.Key];
+                            dictionary[structField.Key] = structField.Value.ConvertToClrType(fieldValue, itemType, options, false);
```

Comment:
Named argument for literal? And in 4 other places below?

Reply:
Done.